lib/likeButton/likeItem.ts

```diff
@@ -0,0 +1,11 @@
+import { arrayUnion, doc, updateDoc } from "firebase/firestore";
+
+import { feedCol } from "@/lib/firestore";
+
+const likeItem = async (userId: string, postId: string) => {
+  await updateDoc(doc(feedCol, postId), {
```

Comment:
```suggestion
  return updateDoc(doc(feedCol, postId), {
```
Return here in case we need to do anything with the response (this should actually auto-turn into a single-line arrow function)